contracts/protocoltoken/CheckpointingToken.sol

```diff
@@ -28,6 +30,8 @@ contract CheckpointingToken is IERC20 {
     // override this function if a totalSupply should be tracked
     function totalSupply()
         public
+        virtual
```

Comment:
in new solidity version we required to explicitly specify if function is overriden and if its virtual ( means it can be overriden)